config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  # get 'home', to: 'home#index', as: 'home'
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  root "home#index"
+
+  resources :financial_groups do
+  resources :financial_entities
+  end
```

Comment:
- [ ] @0sugo Great work on creating your routes, however, you have some routes that are unnecessary, like for updating and deleting categories and entities yet they are not being used.